docs/data/material/getting-started/support/support.md

```diff
@@ -87,35 +70,22 @@ The Tidelift Subscription manages your dependencies for you:
 - Get the tools you need to continuously catalog and understand the open-source software that your application depends on.
 - Your subscription helps pay the open-source community maintainers of the packages you use, to ensure they meet the standards you require.
 - Address issues proactively, with tools that scan for new security, licensing, and maintenance issues, and alert participating open-source maintainers so they can resolve them on your behalf.
-- Tidelift helps measure and improve your open-source dependencies' health – which improves your app's health – and gives a shortlist of high-impact steps your team can take to improve them even more.
+- Tidelift helps measure and improve your open-source dependencies' health—which improves your app's health—and gives a shortlist of high-impact steps your team can take to improve them even more.
 - Get commercial assurances that don't come for free with open-source packages, such as intellectual property indemnification and support under a service level agreement. You expect these guarantees from proprietary software, and you can get them when using open-source as well.
 
-The end result? All of the capabilities you expect from commercial-grade software, for the full breadth of open-source you use. That means less time grappling with esoteric open-source trivia, and more time building your own applications – and your business.
+The end result? All of the capabilities you expect from commercial-grade software, for the full breadth of open-source you use.
+That means less time grappling with esoteric open-source trivia, and more time building your own applications—and your business.
 
 <a
   data-ga-event-category="support"
   data-ga-event-action="tidelift"
   href="https://tidelift.com/subscription/pkg/npm-material-ui?utm_source=npm-material-ui&utm_medium=referral&utm_campaign=enterprise">
-Get more details
+Learn more about Tidelift
 </a>
-
+and
 <a
   data-ga-event-category="support"
   data-ga-event-action="tidelift"
   href="https://tidelift.com/subscription/request-a-demo?utm_source=npm-material-ui&utm_medium=referral&utm_campaign=enterprise">
-Request a demo
-</a>
-
-### Custom work 🛠
```

Comment:
Where do this go? This is here to create a sense of confidence with enterprise, that if they get stuck, and are willing to pay, they will get unlocked.

Reply:
Ah ok, I could've sworn I had a conversation with someone about removing this a long time ago.